ColorSupplier.java

```diff
@@ -0,0 +1,8 @@
+import java.util.Random;
+
+public class ColorSupplier {
+    public Color getRandomColor(int index) {
```

Comment:
```suggestion
    public Color getRandomColor() {
```
You do not need to pass index to this method